updating/index.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/updating-restricted-network-cluster.adoc#update-restricted-network-cluster-update-service[Understanding the {product-title} Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following information:
```

Comment:
[Understanding the {product-title} Update Service] does not match https://docs.openshift.com/container-platform/4.6/updating/understanding-the-update-service.html

Reply:
Changed the xref

Reply:
Addressed